### PR TITLE
Update ItemReference to add information about listItems

### PR DESCRIPTION
**Pull request by k-tsoi**
ListItem resource has references to ItemReference, however ItemReference descriptions only mention driveItem.
Updating ItemReference Resource to also mention listItem

ListItem resource: https://learn.microsoft.com/en-us/graph/api/resources/listitem?view=graph-rest-1.0

### CODE REVIEW

**Review by chackman — CHANGES_REQUESTED** [review]

File: api-reference/v1.0/resources/itemreference.md [hunk 1]
```diff
@@ -41,14 +41,14 @@ Here is a JSON representation of the resource
 
 | Property      | Type              | Description
 |:--------------|:------------------|:-----------------------------------------
-| driveId       | String            | Unique identifier of the drive instance that contains the item. Read-only.
-| driveType     | String            | Identifies the type of drive. See [drive][] resource for values.
-| id            | String            | Unique identifier of the item in the drive. Read-only.
+| driveId       | String            | Unique identifier of the drive instance that contains the driveItem. Only returned if item is located in a [drive][]. Read-only.
+| driveType     | String            | Identifies the type of drive. Only returned if item is location in a [drive][]. See [drive][] resource for values.
```
[the remark this answers is not in the document]
Comment (k-tsoi) [reply api-reference/v1.0/resources/itemreference.md]: updated from location to located in both files

**Review by chackman — APPROVED** [review]

File: api-reference/beta/resources/itemreference.md [hunk 2]
```diff
@@ -40,14 +40,14 @@ Here is a JSON representation of the resource
 
 | Property      | Type              | Description
 |:--------------|:------------------|:-----------------------------------------
-| driveId       | String            | Unique identifier of the drive instance that contains the item. Read-only.
-| driveType     | String            | Identifies the type of drive. See [drive][] resource for values.
-| id            | String            | Unique identifier of the item in the drive. Read-only.
+| driveId       | String            | Unique identifier of the drive instance that contains the driveItem. Only returned if item is located in a [drive][]. Read-only.
```
Comment (Danielabom) [inline api-reference/beta/resources/itemreference.md:43]:
```suggestion
| driveId       | String            | Unique identifier of the drive instance that contains the driveItem. Only returned if the item is located in a [drive][]. Read-only.
```

File: api-reference/beta/resources/itemreference.md [hunk 3]
```diff
@@ -40,14 +40,14 @@ Here is a JSON representation of the resource
 
 | Property      | Type              | Description
 |:--------------|:------------------|:-----------------------------------------
-| driveId       | String            | Unique identifier of the drive instance that contains the item. Read-only.
-| driveType     | String            | Identifies the type of drive. See [drive][] resource for values.
-| id            | String            | Unique identifier of the item in the drive. Read-only.
+| driveId       | String            | Unique identifier of the drive instance that contains the driveItem. Only returned if the item is located in a [drive][]. Read-only.
+| driveType     | String            | Identifies the type of drive. Only returned if item is located in a [drive][].  See [drive][] resource for values.
```
Comment (Danielabom) [inline api-reference/beta/resources/itemreference.md:44]:
```suggestion
| driveType     | String            | Identifies the type of drive. Only returned if the item is located in a [drive][].  See [drive][] resource for values.
```

File: api-reference/v1.0/resources/itemreference.md [hunk 4]
```diff
@@ -41,14 +41,14 @@ Here is a JSON representation of the resource
 
 | Property      | Type              | Description
 |:--------------|:------------------|:-----------------------------------------
-| driveId       | String            | Unique identifier of the drive instance that contains the item. Read-only.
-| driveType     | String            | Identifies the type of drive. See [drive][] resource for values.
-| id            | String            | Unique identifier of the item in the drive. Read-only.
+| driveId       | String            | Unique identifier of the drive instance that contains the driveItem. Only returned if item is located in a [drive][]. Read-only.
```
Comment (Danielabom) [inline api-reference/v1.0/resources/itemreference.md:44]:
```suggestion
| driveId       | String            | Unique identifier of the drive instance that contains the driveItem. Only returned if the item is located in a [drive][]. Read-only.
```

File: api-reference/v1.0/resources/itemreference.md [hunk 5]
```diff
@@ -41,14 +41,14 @@ Here is a JSON representation of the resource
 
 | Property      | Type              | Description
 |:--------------|:------------------|:-----------------------------------------
-| driveId       | String            | Unique identifier of the drive instance that contains the item. Read-only.
-| driveType     | String            | Identifies the type of drive. See [drive][] resource for values.
-| id            | String            | Unique identifier of the item in the drive. Read-only.
+| driveId       | String            | Unique identifier of the drive instance that contains the driveItem. Only returned if the item is located in a [drive][]. Read-only.
+| driveType     | String            | Identifies the type of drive. Only returned if item is located in a [drive][]. See [drive][] resource for values.
```
Comment (Danielabom) [inline api-reference/v1.0/resources/itemreference.md:45]:
```suggestion
| driveType     | String            | Identifies the type of drive. Only returned if the item is located in a [drive][]. See [drive][] resource for values.
```

**Review by Danielabom — APPROVED** [review]
Thank you for working on this PR @k-tsoi. PR approved.

**Pull request merged**